### PR TITLE
Documentation 

### PR DESCRIPTION
**Pull request by jacraul**
### Pull Request Overview

Changes in the GitHub project link.

### Author

Signed-off-by: Raul-Anton Jac <raul_anton.jac@stud.fils.upb.ro>

### CODE REVIEW

File: docs/project/2024/raul_anton.jac/index.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
+# MeloLED
+MP3 Player with LED strip using Pico RP2040
+
+:::info 
+
+**Author**: Raul-Anton Jac \
+**GitHub Project Link**: [https://github.com/jacraul/meloled](https://github.com/jacraul/meloled)
```
Comment (alexandruradovici) [inline docs/project/2024/raul_anton.jac/index.md:7]:
```suggestion
**GitHub Project Link**: https://github.com/jacraul/meloled
```

I don't think this is the project's repository, it has to be the one that was created when you accepted the Project assignment.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

**Comment by alexandruradovici** [comment]
Please fill the pull request template.

File: docs/project/2024/raul_anton.jac/index.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
+# MeloLED
+MP3 Player with LED strip using Pico RP2040
+
+:::info 
+
+**Author**: Raul-Anton Jac \
+**GitHub Project Link**: [https://github.com/UPB-FILS-MA/project-jacraul](https://github.com/UPB-FILS-MA/project-jacraul)
```
Comment (alexandruradovici) [inline docs/project/2024/raul_anton.jac/index.md:7]:
```suggestion
**GitHub Project Link**: https://github.com/UPB-FILS-MA/project-jacraul
```

**Review by alexandruradovici — CHANGES_REQUESTED** [review]

**Review by alexandruradovici — APPROVED** [review]

**Comment by alexandruradovici** [comment]
Please fill the pull request template.

**Review by alexandruradovici — CHANGES_REQUESTED** [review]
Please fill the pull request template.

**Comment by jacraul** [comment]
I guess it's okay now

**Review by alexandruradovici — APPROVED** [review]

**Review by irina-nita — APPROVED** [review]

**Review by ZamfirAndreiPaul — APPROVED** [review]

**Pull request merged**